schema/cassandra/temporal/schema.cql

```diff
@@ -184,3 +184,28 @@ CREATE TABLE cluster_membership
 
 CREATE INDEX cm_lastheartbeat_idx on cluster_membership (last_heartbeat);
 CREATE INDEX cm_sessionstart_idx on cluster_membership (session_start);
+
+CREATE TABLE queues
+(
+    queue_type        int,
+    queue_name        text,
+    metadata_payload  blob,
+    metadata_encoding text,
+    version           bigint,
+    PRIMARY KEY ((queue_type, queue_name))
```

Comment:
let's have a quick sync if `queue_partition` should be part of the PRIMARY KEY as well.

Reply:
synced offline. it's probably unnecessary and will bring issues like how/where to store information for the entire queue. and we are not planning to use this implementation for very high throughput use case anyway.